### PR TITLE
Fix dependencies

### PR DESCRIPTION
**Pull request by helion-du-mas-des-bourboux-thales**
Fixes https://github.com/fumihwh/onnx-pytorch/issues/34

### CODE REVIEW

**Comment by helion-du-mas-des-bourboux-thales** [comment]
@fumihwh This PR is ready to be merged, and passes all checks. It get the proper packages requirements,
sorted the requirements, and passes from requirements.txt to setup.py.

File: requirements.txt [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
-torch==1.10.0
-typing-extensions==3.7.4.3
+numpy>=1.19.5
+onnx>=1.10.2
+onnxruntime>=1.9.0
+pytest>=6.2.5
+PyYAML>=6.0
+setuptools>=59.2.0
+sympy>=1.9
+torch>=1.10.0
```
Comment (fumihwh) [inline requirements.txt:8]: If you change the version here, you should edit corresponding version in `README.md:L8`

**Comment by fumihwh** [comment]
More, if we want back compatibility, we should consider to add `torch==1.8.1` and `onnx==1.9.0` to CI.
Let's think more on this. Just fix `README.md` in this PR.
And I am also considering to use python reflect to handle version compatibility, avoiding pytorch API change (maybe they won't).

**Comment by helion-du-mas-des-bourboux-thales** [comment]
@fumihwh I just fixed the versions in README.md
Do you want me to do something else?

**Review by fumihwh — APPROVED** [review]

**Comment by fumihwh** [comment]
Thanks! I remove the sentence for now.

**Pull request merged**